gocourse03/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"math/rand/v2"
+	"slices"
+	"time"
+)
+
+func main() {
+	rodents := []Rodent{
+		NewRodent(11, RodentRat, FromTo{}, NewDailyMovements()),
+		NewRodent(12, RodentRat, FromTo{}, NewDailyMovements()),
+		NewRodent(13, RodentRat, FromTo{}, NewDailyMovements()),
+		NewRodent(66, "To delete type", FromTo{}, NewDailyMovements()),
```

Comment:
Ось так має працювати:
```suggestion
		NewRodent(11, RodentRat, FromTo{}, nil),
		NewRodent(12, RodentRat, FromTo{}, nil),
		NewRodent(13, RodentRat, FromTo{}, nil),
		NewRodent(66, "To delete type", FromTo{}, nil),
```

`NewDailyMovements` можна взагалі видалити.

В Go є такий принцип "Make zero value useful". https://dave.cheney.net/2013/01/19/what-is-the-zero-value-and-why-is-it-useful

В нашому випадку можемо використовувати `nil` як корисне "zero value".